### PR TITLE
Support gw1ns 2c

### PR DESCRIPTION
**Pull request by trabucayre**
This PR add support for GW1NS-2C FPGA:
The first commit avoid a crash like:
```bash
  File "/xxx/apicula/apycula/gowin_pack.py", line 91, in place
    bits = tiledata.bels['BANK'].modes['DEFAULT']
KeyError: 'BANK'
make: *** [Makefile:27 : blinky-honeycomb.fs] Erreur 1
```
Second commit add everything required to generate the pickle for this device and update readme (Dockerfile is partially updated -> needs to store xls somewhere)
And finally last commit add the honeycomb in examples

### CODE REVIEW

File: apycula/chipdb.py [hunk 1]
```diff
@@ -165,6 +168,10 @@ def xls_pinout(family):
         return {
             "GW1N-4": get_pins("GW1N-4"),
         }
+    elif family == "GW1NS-2":
+        return {
+            "GW1NS-2C": get_pins("GW1NS-2C"),
```
Comment (pepijndevos) [inline apycula/chipdb.py:173]: No `C` here, right?

File: apycula/tiled_fuzzer.py [hunk 2]
```diff
@@ -31,6 +31,12 @@
 device = sys.argv[1]
 
 params = {
+    "GW1NS-2": {
+        "package": "QN48",
```
Comment (pepijndevos) [inline apycula/tiled_fuzzer.py:35]: Usually it's best to take the package with the most pins here, this speeds up fuzzing.

**Comment by pepijndevos** [comment]
I have uploaded https://wishfulcoding.nl/gowin/UG825-1.2.1E_GW1NS-2C%20Pinout.xlsx and https://wishfulcoding.nl/gowin/UG822-1.2.1E_GW1NS-2%20Pinout.xlsx so please add these to the dockerfile.

Awesome work!!

File: apycula/chipdb.py (hunk 1 above)
Comment (pepijndevos) [reply apycula/chipdb.py:173]: Or rather... I think we should add both the C and non-C variants. They can be supported by the same chipdb, it's only the pins that get added.

Comment (pepijndevos) [reply apycula/chipdb.py:173]: Like with the GW1N-9 and GW1NR-9, same bitstream, different pinouts.

Comment (trabucayre) [reply apycula/chipdb.py:177]: There is no GW1NS-2C directory in IDE/share so yes need to use GW1NS-2
Pushed GW1NS-2 variant for pinout

File: apycula/tiled_fuzzer.py (hunk 2 above)
Comment (trabucayre) [reply apycula/tiled_fuzzer.py]: Done

**Comment by trabucayre** [comment]
Dockerfile updated too
Thanks

**Comment by pepijndevos** [comment]
Great!! I'll do a release once CI is done.

**Pull request merged**